packages/shared/lib/models/Proxy.ts

```diff
@@ -45,6 +45,8 @@ export interface ApplicationConstructedProxyConfiguration extends BaseProxyConfi
     token: string | BasicApiCredentials | ApiKeyCredentials | AppCredentials | TbaCredentials | TableauCredentials;
     provider: Provider;
     connection: Connection;
+    sessionId?: string;
+    devKey?: string;
```

Comment:
do we really need to add those 2 attributes which are specific to a single provider?

Reply:
I would try to keep the ApplicationConstructedProxyConfiguration as generic as possible. sessionId and devKey are already part of of the `BillCredentials`. Can we get them from there?